### ISSUE TITLE
doesn't seem to work

### ISSUE THREAD

**Issue by airdrummer** [nrkbeta/nrkbetaquiz#10]
added simple question in editor, but no quiz @ https://cardinalglen.org/spring-arrives/
perhaps login required is interfering?

**Comment by serialc** [comment]
Have you tried logging in? Seems unlikely this plugin will display if comments are disabled until you log in.

**Comment by airdrummer** [comment]
yes, comments are available to logged in users, no quiz is displayed
